classes/02class/exercises/c02-network07/zarajoy/README.md

```diff
@@ -0,0 +1,445 @@
+    "AssociationState": {
+        "State": "associated"
+    }
+}
+z@bacon:~$ aws ec2 associate-route-table --route-table-id rtb-07a9d3ac37a9cb8e0 --subnet-id subnet-0abbcb057289b670d
+{
+    "AssociationId": "rtbassoc-0e023ca49445d24bf",
+    "AssociationState": {
+        "State": "associated"
+    }
+}
+z@bacon:~$ aws ec2 associate-route-table --route-table-id rtb-07a9d3ac37a9cb8e0 --subnet-id subnet-00649d5b0632cc2be
+{
+    "AssociationId": "rtbassoc-0bcdf6dae4c22038c",
+    "AssociationState": {
+        "State": "associated"
+    }
+}
+z@bacon:~$ aws ec2 describe-route-tables --route-table-ids rtb-07a9d3ac37a9cb8e0
+{
+    "RouteTables": [
+        {
+            "Associations": [
+                {
+                    "Main": false,
+                    "RouteTableAssociationId": "rtbassoc-0e023ca49445d24bf",
+                    "RouteTableId": "rtb-07a9d3ac37a9cb8e0",
+                    "SubnetId": "subnet-0abbcb057289b670d",
+                    "AssociationState": {
+                        "State": "associated"
+                    }
+                },
+                {
+                    "Main": false,
+                    "RouteTableAssociationId": "rtbassoc-0bcdf6dae4c22038c",
+                    "RouteTableId": "rtb-07a9d3ac37a9cb8e0",
+                    "SubnetId": "subnet-00649d5b0632cc2be",
+                    "AssociationState": {
+                        "State": "associated"
+                    }
+                },
+                {
+                    "Main": false,
+                    "RouteTableAssociationId": "rtbassoc-0fef00ef8a9b54683",
+                    "RouteTableId": "rtb-07a9d3ac37a9cb8e0",
+                    "SubnetId": "subnet-0907f22726c996fd5",
+                    "AssociationState": {
+                        "State": "associated"
+                    }
+                }
+            ],
+            "PropagatingVgws": [],
+            "RouteTableId": "rtb-07a9d3ac37a9cb8e0",
+            "Routes": [
+                {
+                    "DestinationCidrBlock": "192.168.0.0/16",
+                    "GatewayId": "local",
+                    "Origin": "CreateRouteTable",
+                    "State": "active"
+                }
+            ],
+            "Tags": [
+                {
+                    "Key": "Name",
+                    "Value": "devopsacademy-rt-public"
+                }
+            ],
+            "VpcId": "vpc-09d2f2719d50d1f7f",
+            "OwnerId": "512742231244"
+        }
+    ]
+}
+```
+
+- Commands for creating the following routes:
+
+|route table|destination|target|
+|-|-|-|
+|devopsacademy-rt-public|0.0.0.0|`devopsacademy-igw`|
+|devopsacademy-rt-private|0.0.0.0|`devopsacademy-ngw`|
+
+```
+z@bacon:~$ aws ec2 create-route  --route-table-id rtb-07a9d3ac37a9cb8e0  --destination-cidr-block 0.0.0.0/0 --gateway-id igw-062abdced9341d7f8
+{
+    "Return": true
+}
+#redo c02-network06 as it states to delete NAT gateway once exercise is complete...
```

Comment:
Yeah, it is used here again 😆  
Better to create again than paying the bill

Reply:
plus extra practise :D